src/libraries/System.Numerics.Tensors/tests/TensorPrimitives.Generic.cs

```diff
@@ -374,15 +374,14 @@ public static IEnumerable<object[]> SpanDestinationFunctionsToTest()
             yield return Create(TensorPrimitives.DegreesToRadians, T.DegreesToRadians);
             yield return Create(TensorPrimitives.Exp, T.Exp);
             // TODO https://github.com/dotnet/runtime/issues/98861
-            yield return Create(TensorPrimitives.Exp2, T.Exp2, Helpers.DetermineTolerance<T>(doubleTolerance: 1e-6, floatTolerance: 1e-5f));
```

Comment:
Do we still need these todos?

Reply:
I figured I'd keep them so long as the tolerances are above the scalar baselines. In practice we could decide is that `1e-14` is a good-enough baseline at which point most of these can go away.